File: Coding Projects/C++/Section 10/RPGProject/Warrior.cpp
//
// Created by dbehl on 5/14/2022.
//
#include "Player.h"
#include "Warrior.h"

Warrior::Warrior(string name, race Race) : Player(name, Race, 200, 0) {
}

string Warrior::attack() const {
    return "I will destroy you with my sword, foul demon!";
}

File: Coding Projects/My Projects/delhi/delhi/Warrior.cs
using System;
namespace rpg
{
	public class Warrior : Player
	{
		public Warrior(string name, race Race) : base(name, Race, 200, 0) {}
		
		override public string attack()
        {
			return "I will destroy you with my sword, foul demon!";
		}
	}
}

